lua/gp/config.lua

```diff
@@ -120,7 +107,7 @@ local config = {
 			-- string with model name or table with model name and parameters
 			model = { model = "gpt-3.5-turbo", temperature = 1.1, top_p = 1 },
 			-- system prompt (use this to specify the persona/role of the AI)
-			system_prompt = default_chat_system_prompt,
+			system_prompt = defaults.default_chat_system_prompt,
```

Comment:
how about inlined require to make it copy pastable for people without the need to look for `defaults`

and yes, I would drop the redundant prefix

```lua
system_prompt = require("gp.defaults").chat_system_prompt,
```

Reply:
done